docs/userguide/constraints.md

```diff
@@ -222,10 +233,123 @@ DiscreteLinkedParametersConstraint(
 | 3 | Octanol           | Octanol             |                                         |
 
 ### ``DiscreteDependenciesConstraint``
-Content coming soon...
+A dependency is a situation where parameters depend on other parameters.
+Let's say an experimental setup has a parameter called "Switch" which turns on 
+pieces of equipment that are optional.
+This means the other parameters (called ``affected_parameters``) are only relevant if 
+the "Switch" parameter has the value "on". If the switch is "off", the affected parameters 
+are irrelevant.
+
+You can specify such a dependency with the ``DiscreteDependenciesConstraint``, which 
```

Comment:
General question: Should lines like this, where a Constraint or "technical term" is mentioned the first time contain a link to the documentation of the corresponding class?

Reply:
a good idea I will try and include where relevant

Reply:
included here and also over the course of resolving the no ticks in headlines question